Courses/Navigation/NavigationBar.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  NavigationBar.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 11.10.2022.
+//
+
+import SwiftUI
+
+struct NavigationBar: View {
+    var title = ""
+    @Binding var hasScrolled: Bool
```

Comment:
This name still doesn't make a lot of sense

Reply:
@yura-hb , ah, missed it, fixed.